spec/classes/nodejs_spec.rb

```diff
@@ -99,4 +100,24 @@
       .with_content(/(.*)NODE_PATH=\/usr\/local\/node\/node-v5.4.1\/lib\/node_modules/)
     }
   end
+
+  describe 'default parameters with use_cpus set manually to 1' do
+    let(:params) {{
+      :use_cpus => 1
+    }}
+
+    it { should contain_nodejs__install('nodejs-stable') \
+      .with_version('v6.0.0') \
+      .with_target_dir('/usr/local/bin') \
+      .with_make_install('true') \
+      .with_use_cpus(1)
+    }
+
+    it { should contain_file('/usr/local/node/node-default') \
```

Comment:
this assertion can be removed as it is not relevant for your special case (it's good enough to assert against things that should have a special behavior in a test case)